### ISSUE TITLE
Convolution layer setting when learn from scratch

### ISSUE THREAD

**Issue by mjohn123** [liuzhuang13/DenseNetCaffe#10]
Hi, I am using your prototxt to learn from scratch, instead of fine-tuning with ImageNet. The reason is that my dataset is totally different with ImageNet.

Looking at your prototxt, I found that your convolution layer does not have the `lr_mult` setting [here](https://github.com/liuzhuang13/DenseNetCaffe/blob/master/make_densenet.py#L10)
```
conv = L.Convolution(relu, kernel_size=ks, stride=stride, 
                    num_output=nout, pad=pad, bias_term=False, 
                    weight_filler=dict(type='msra'), bias_filler=dict(type='constant'))
```
I think it must be add the lr_mult as follows:
```
conv = L.Convolution(relu, kernel_size=ks, stride=stride, 
                    num_output=nout, pad=pad, bias_term=False, 
                    weight_filler=dict(type='msra'), bias_filler=dict(type='constant', value=0),
                    param=[dict(lr_mult=1, decay_mult=1))
```
Am I right?

**Comment by liuzhuang13** [comment]
Hi! I think the default lr_multi is 1, so there won't be a difference. Thanks

**Comment by John1231983** [comment]
Thanks. One more thing, why did not learn bias_term of Convolution ( bias_term: false)? I knew that ResNet also does not learn it, and the number of the parameter will increase when we learn it. But it will increase the performance. Any reason for ignoring of bias learning

**Comment by mjohn123** [comment]
Thanks. One more thing, why did not learn bias_term of Convolution layer ( bias_term: false)? I knew that Resnet also did not learn it. However, the performance may increase when we learn it, although the number of the parameter will increase.  Do we have any reason for ignoring the learning bias_term?

**Comment by liuzhuang13** [comment]
The reason is that the following batch normalization operation will completely cancel the effect of that bias term.
